docs/changehistory/NextVersion.md

```diff
@@ -125,6 +125,21 @@ New static type guards `Parser.isParsedQuantity` and `Parser.isParseError` can b
 
 The alpha UnitConversionSpec interface now requires a "system" property that can be used during parsing to help determine the unit to parse the value.
 
+### UI-abstract
+
+Property `onClick` in `LinkElementsInfo` was changed to be mandatory. Now it is easier to work with with the mentioned interface.
+Only problem that might happen is if we want to setup a custom `LinkElementsInfo.matcher` while still using default `LinkElementsInfo.onClick`.
+
+In order to fix this problem, specify default onClick method when creating `LinkElementsInfo` which can be found in: `@bentley\ui-components\propertygrid\component\PropertyGridCommons.getLinks`
+Example:
+
+```ts
+const exampleLinksElementInfo: LinkElementsInfo = {
+  onClick: PropertyGridCommons.getLinks,
+  matcher: CUSTOM_MATCHER,
+}
+```
+
```

Comment:
We can't recommend using `@internal` APIs...

~~~suggestion
### ui-abstract package

Property `onClick` in [LinkElementsInfo]($ui-abstract) was changed to be mandatory. Also, the first [PropertyRecord]($ui-abstract) argument was removed from the method. Suggested ways to resolve:

- If you have a function `myFunction(record: PropertyRecord, text: string)` and use the first argument, the issue can be resolved with a lambda:

  ```ts
  record.links = {
    onClick: (text) => myFunction(record, text),
  };
  ```

- If you were omitting the `onClick` method to get the default behavior, it can still be achieved by not setting `PropertyRecord.links` at all. It's only valid to expect default click behavior when default matcher is used, but if a custom matcher is used, then the click handled can be as simple as this:

  ```ts
  record.links = {
    onClick: (text) => { window.open(text, "_blank"); },
  };
  ```


~~~